sdk/core/core-httpclients/test/browser/xhrHttpClient.spec.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert, describe, beforeEach, afterEach, it, vi } from "vitest";
+import { createXhrHttpClient } from "../../src/xhrHttpClient";
+import { createPipelineRequest } from "@azure/core-rest-pipeline";
+
+describe("XhrHttpClient", function () {
```

Comment:
Will migrate tests in a future PR.  Since the xhrHttpClient code is existing without new changes, it should be safe.